polytope_server/common/staging/s3_staging.py

```diff
@@ -126,9 +129,19 @@ def create(self, name, data, content_type):
             self.client._complete_multipart_upload(self.bucket, name, upload_id, parts)
         except Exception:
             self.client._remove_incomplete_upload(self.bucket, name, upload_id)
+            end = timeit.default_timer()
+            delta = timedelta(seconds=(end - start)).total_seconds()
```

Comment:
The timedelta object is not really needed in this situation.

Reply:
I removed the timedelta